script/load_schema.sh

```diff
@@ -13,3 +13,5 @@ if [ -z "$DB_VERSION" ] || [ "$DB_VERSION" != "$FILE_VERSION" ]; then
 else
   echo "✅ Schema is up to date (version $DB_VERSION)."
 fi
+
+####################
```

Comment:
I don't understand this commit. It looks like the previous CI run failed randomly because of an external error. Can you remove this commit again?